src/assets/stylesheets/_config.scss

```diff
@@ -75,6 +79,54 @@ $md-color-white--lighter:             hsla(0, 0%, 100%, 0.3) !default;
 $md-color-white--lightest:            hsla(0, 0%, 100%, 0.12) !default;
 $md-color-white--transparent:         hsla(0, 0%, 100%, 0) !default;
 
+// ----------------------------------------------------------------------------
+// CSS Variables: base colors
+// ----------------------------------------------------------------------------
+
+:root {
+  // Primary and accent colors
+  --md-color-primary:                         #{$md-color-primary};
+  --md-color-primary-values:                  #{hex2rgb($md-color-primary)};
```

Comment:
This looks like a smell to me. Where do we need those values? For different opacities? We should define those color-opacity combinations as variables.

Reply:
Here https://github.com/squidfunk/mkdocs-material/pull/1409/files#diff-5c58c014d3bb6c0f3026ce46fab89998R126 to replace opacify()

Reply:
I'll check out your changes when I'm back and will take care of merging. Thanks again for your support!

Reply:
I will proceed on this PR when I finished all the JavaScript migration. Sorry for the delay!